src/Sulu/Bundle/ActivityBundle/Infrastructure/Sulu/Metadata/ActivityListMetadataProvider.php

```diff
@@ -0,0 +1,52 @@
+<?php
+
+/*
+ * This file is part of Sulu.
+ *
+ * (c) Sulu GmbH
+ *
+ * This source file is subject to the MIT license that is bundled
+ * with this source code in the file LICENSE.
+ */
+
+namespace Sulu\Bundle\ActivityBundle\Infrastructure\Sulu\Metadata;
+
+use Sulu\Bundle\AdminBundle\Metadata\ListMetadata\ListMetadata;
+use Sulu\Bundle\AdminBundle\Metadata\MetadataInterface;
+use Sulu\Bundle\AdminBundle\Metadata\MetadataProviderInterface;
+
+class ActivityListMetadataProvider implements MetadataProviderInterface
```

Comment:
think i would use `activities` to match the list key? 

```suggestion
class ActivitesListMetadataProvider implements MetadataProviderInterface
```

Reply:
i would prefer to implement an extension point for this, but i think we need to do that after the release candidate. would add a `@internal` doc